scripts/status-report.sh

```diff
@@ -0,0 +1,228 @@
+#!/usr/bin/env bash
+#set -x
+
+if [ -z "${COMPOSE_PROJECT_NAME:-}" ]; then
+  # shellcheck disable=SC1091
+  source .env
+fi
+
+# Report ID used to identify queries generated by this script,
+# and avoid cache issues
+report_id=$(date +%s)
+# If an error is found, set to 1
+# This will be used as script exit code
+errors_found=0
+
+# Compare expected `make status` output with current instance
+function check_status() {
+  local status
+  local expected
+
+  status=$(make status)
+  expected="
+*** .env file ***
+
+# docker-compose env variables
+COMPOSE_FILE=${COMPOSE_FILE}
+COMPOSE_PROJECT_NAME=${COMPOSE_PROJECT_NAME}
+
+*** Docker compose status ***
+
+              Name                            Command               State                                Ports
+--------------------------------------------------------------------------------------------------------------------------------------------
+${COMPOSE_PROJECT_NAME}_db_1          docker-entrypoint.sh mysqld      Up      3306/tcp, 33060/tcp
+${COMPOSE_PROJECT_NAME}_node_1        docker-entrypoint.sh node        Up
+${COMPOSE_PROJECT_NAME}_openresty_1   /app/bin/entrypoint.sh /sb ...   Up      443/tcp, 80/tcp
+${COMPOSE_PROJECT_NAME}_php-fpm_1     /app/bin/entrypoint.sh /sb ...   Up      9000/tcp
+${COMPOSE_PROJECT_NAME}_redis_1       docker-entrypoint.sh redis ...   Up      6379/tcp
+${COMPOSE_PROJECT_NAME}_traefik_1     /traefik --web --docker -- ...   Up      0.0.0.0:443->443/tcp, 0.0.0.0:80->80/tcp, 0.0.0.0:8080->8080/tcp
+
+*** Credentials ***
+
+             | User      | Password
+-----------------------------------------------
+ Wordpress   | admin     | admin
+-----------------------------------------------
+ Database    | planet4   | Phoh4zaeshaiT9ee
+             | root      | root
+
+*** Links ***
+
+ Frontend   - http://www.planet4.test
+ Backend    - http://www.planet4.test/admin
+ Traefik    - http://localhost:8080/
+"
+
+  # Ignore all spaces, empty lines and separator lines
+  if ! diff_result=$(diff -wbB --color -I '^---.*' <(echo "${expected}" ) <(echo "${status}")); then
```

Comment:
Can you also pass this through shfmt for some minor tweaks?

`shfmt -i 2 -ci -d scripts/status-report.sh`

and this will auto-fix the issues:

`shfmt -i 2 -ci -w scripts/status-report.sh`